src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -557,6 +557,7 @@ internal SomeAttribute()
 
     static SomeAttribute()
     {
+        Console.WriteLine();
```

Comment:
What is the purpose of this change?

Reply:
artifact of the approach which excluded empty user-defined static constructors from emit. reverting.